backend/utils/sidebar.py

```diff
@@ -38,7 +38,11 @@ async def __get_room_members(
         for room_member_id in room_members.keys():
             member_data = await DB.get_member(room_member_id, org_members)
             username = member_data.get("user_name", "no user name")
-            image_url = member_data.get("image_url", DEFAULT_DM_IMG)
+            image_url = (
+                DEFAULT_DM_IMG
+                if member_data.get("image_url") == ""
+                else member_data.get("image_url")
+            )
```

Comment:
Here, we could also use a boolean exp. It's more concise.
```suggestion
            image_url = member_data.get("image_url") or DEFAULT_DM_IMG
```

Reply:
Yea
@mykie88 please kindly look at @zxenonx suggestion

Reply:
Thanks a lot bosses. The suggestion worked like a charm.